### PR TITLE
Add documentation for xslURL in sitemap package

### PR DESCRIPTION
**Pull request by sondr3**
#### Description

Add documentation for the new [`xslURL` in `@astrojs/sitemap`](https://github.com/withastro/astro/pull/11485).

### CODE REVIEW

**Comment by sarah11918** [comment]
Hi @sondr3 ! Can you please fill out the PR template and add a description so that I know what this is referring to? Thank you!

**Review by sarah11918 — COMMENTED** [review]
Thank you for this contribution, @sondr3 ! The formatting and structure all looks great and you've integrated this content nicely into existing docs!

Just made a tiny edit to make the content style more consistent with our docs! :raised_hands: I will also need to wait on platform to approve and release the feature before docs can be merged, so this PR is going to get a bunch of labels to help keep me organized. :smile:

**Comment by sarah11918** [comment]
Well, this is odd! @sondr3 I know I made an edit here originally, but it's probably not saved on GitHub. Is there any chance my commit still exists in your history somewhere?

I can re-edit to conform to our standards, but if I already did this once, it would be handy if that could just be applied here!

**Comment by sondr3** [comment]
I checked on my machine and I don't have anything besides my own commit sadly 😢

**Comment by sarah11918** [comment]
@sondr3 No worries! Thank you for checking! I'll make some tiny edits today to just conform to our style, but wanted to wait just in case there was a history of what I'd already done!

File: src/content/docs/en/guides/integrations-guide/sitemap.mdx [hunk 1]
```diff
@@ -401,6 +401,28 @@ The resulting sitemap looks like this:
 ...
 ```
 
+### xslURL
+
+This sets the URL of an XSL stylesheet that enables you to style and prettify your sitemap. An example of a XSL stylesheets for sitemaps can be found [here](https://github.com/pedroborges/xml-sitemap-stylesheet).
+
+:::note
+The URL can be either relative or absolute. If it is relative, it will be resolved relative to the `site` URL.
+:::
```
Comment (sarah11918) [inline src/content/docs/en/guides/integrations-guide/sitemap.mdx:410]:
```suggestion
The URL of an XSL stylesheet to style and prettify your sitemap. 

The value set can be either a path relative to your configured `site` URL for a local stylesheet, or can be an absolute URL link to an external stylesheet.
```
This linked to a package with instructions that wasn't the same usage as what was described here, so I think just pointing people there isn't as helpful as providing JUST a URL that's an example they can drop in and test with.

So I think we'll just remove that for now, but I'd be open to (even a follow up PR) adding a good link to a stylesheet people can point to directly (not a package): is there one people can just drop in and try immediately? Is there some known directory people can browse somewhere? Is this a common thing people showcase anywhere?

So, doesn't need to block THIS PR, but would be happy to have an addition to it that helps people discover existing stylesheets, or learn how to write their own!

**Review by sarah11918 — COMMENTED** [review]
Thank you for your patience @sondr3 and @florian-lefebvre !

With some minor tweaking below, this is (once again?) perfectly fine! 😅 

See my comment below re: a follow up. If there's something you want to sneak in before merging, it's fine by me. If this needs to merge more quickly, then it can be done as an add-on at any later point!

PS: I have committed directly these changes because it fixes our link error, but am still open to discussion if needed!

**Comment by sarah11918** [comment]
Looks like this was released!

**Pull request merged**